lib/ITip/Broker.php

```diff
@@ -333,7 +333,7 @@ protected function processMessageReply(Message $itipMessage, ?VCalendar $existin
 
         // Finding all the instances the attendee replied to.
         foreach ($itipMessage->message->VEVENT as $vevent) {
-            $recurId = isset($vevent->{'RECURRENCE-ID'}) ? $vevent->{'RECURRENCE-ID'}->getValue() : 'master';
+            $recurId = isset($vevent->{'RECURRENCE-ID'}) ? $vevent->{'RECURRENCE-ID'}->getDateTime()->getTimestamp() : 'master';
```

Comment:
Note for myself ( and maybe helps others):
https://www.php.net/manual/en/datetime.gettimestamp.php returns the Unix timestamp (seconds since "Unix zero time").

So that is a unique number that corresponds to the day-and-time of the particular event. If that timestamp of a recurrence matches the timestamp of a reply, then we know that it is a reply to that occurrence of the event - seems good to me.

Reply:
Yes, the unix timestamp avoids the timezone problem

Reply:
I pushed a commit that adds comments about the use of getTimestamp()